pkg/apiserver/user/verify_sql_user.go

```diff
@@ -59,14 +60,20 @@ func VerifySQLUser(tidbClient *tidb.Client, userName, password string) (writeabl
 	if err != nil {
 		return false, err
 	}
-	// 2. Get grants
+	// 2. Check SkipGrantTable
+	// Note: Currently, if TiDB enable the skip-grant-table, running `show grants` will get error.
+	// So this is a workaround before the above bug is fixed.
```

Comment:
Does this mean that the non-root user has more privileges in this case? What is the impact on the tidb-dashboard?

Reply:
yes, if someone is connected to the tidb-server which `skip-grant-table=true`, he can do anything